docs/src/pages/[platform]/theming/style-props/examples/ColorExamples.tsx

```diff
@@ -0,0 +1,27 @@
+import { Alert, useTheme } from '@aws-amplify/ui-react';
+
+// use any CSS value
+export const ColorStylePropExample = () => {
+  return (
+    <Alert variation="info" color="rgb(125, 214, 232)">
```

Comment:
My suggestion here to make this look good on both dark and light mode is to add a background color as white and set the color value to the same as `brand.primary.80` below.
```suggestion
    <Alert backgroundColor="#fff" color="hsl(190, 95%, 30%)">
```

![image](https://user-images.githubusercontent.com/6165315/174460819-71fda311-bac3-44a2-92f4-995cacd9cea8.png)

![image](https://user-images.githubusercontent.com/6165315/174460825-d2a68898-e5d9-4ac5-be39-c0b1d74ca74f.png)

Reply:
thanks this has been changed, but instead of using primary 80, I went with primary 60 so that it is the same color on light and dark mode